models/schemas/Entry.js

```diff
@@ -0,0 +1,11 @@
+class Entry {
+  constructor(code, name, room, len, section) {
+    this.courseCode = code;
+    this.courseName = name;
+    this.sectionRoom = room;
+    this.numHours = len;
+    this.section = section;
+  }
+}
+
+module.export = Entry;
```

Comment:
Why are these redeclared? Is it not possible to access what is inside `/client` from here?

Reply:
yes